packages/react/src/internal/useIdPrefix.js

```diff
@@ -7,7 +7,7 @@
 
 import React from 'react';
 
-export const IdPrefixContext = React.createContext(null);
+export const IdPrefixContext = React.createContext('cds');
```

Comment:
@tay1orjones do we want to update this to `null` I'm doing it here, but want to double check before committing

Reply:
No I think we need to keep it as `null`. IIRC this is because we weren't configuring an ID prefix by default before `useIdPrefix` was added. 

Adding a prefix after the fact like this would probably need to be a breaking change

Reply:
@andreancardona No I think we need to keep it as `null`. IIRC this is because we weren't configuring an ID prefix by default before `useIdPrefix` was added. 

Adding a prefix after the fact like this would probably need to be a breaking change